content/en/blog/2024/otel-collector-anti-patterns/index.md

```diff
@@ -0,0 +1,176 @@
+not good enough. Wouldn’t it be nice to know when one of your Collectors is
+malfunctioning, or when data is being dropped? That way, you can take action
+before things start to escalate. This is where monitoring your Collectors can be
+very useful.
+
+But how does one monitor a Collector? The OTel Collector already emits
+[metrics for the purposes of its own monitoring](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/monitoring.md).
+These can then be sent to your Observability backend for monitoring.
+
+### 4- Not using the right Collector Distribution (or not building your own distribution)
+
+There are two official distributions of the OpenTelemetry Collector:
+[Core](https://github.com/open-telemetry/opentelemetry-collector), and
+[Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib).
+
+The Core distribution is a bare-bones distribution of the Collector for OTel
+developers to develop and test. It contains a base set of
+[extensions](/docs/collector/configuration/#service-extensions),
+[connectors](/docs/collector/configuration/#connectors),
+[receivers](/docs/collector/configuration/#receivers),
+[processors](/docs/collector/configuration/#processors), and
+[exporters](/docs/collector/configuration/#exporters).
+
+The Contrib distribution is for non-OTel developers to experiment and learn. It
+also extends the Core distribution, and includes components created by
+third-parties (including vendors and individual community members), that are
+useful to the OpenTelemetry community at large.
+
+Neither Core nor Contrib is meant to be part of your production workload. Core
+is too bare-bones and wouldn’t suit an organization’s needs. Many OpenTelemetry
+practitioners, however, deploy Contrib in their respective organizations. But
+Contrib has many components, and you likely won’t need every single exporter,
+receiver, processor, connector, and extension. That would be overkill, and your
+Collector instance ends up needlessly bloated.
+
+But how do you pick and choose the components that you need? The answer is to
+build your own distribution, and you can do that using a tool called the
+[OpenTelemetry Collector Builder](/docs/collector/custom-collector/) (OCB). In
+addition, at some point, you may need to create your own custom Collector
+component, such as a processor or exporter. The OCB allows you to integrate your
+custom components AND pick and choose the Contrib components that you need.
+
+It is also worth mentioning that some vendors build their own Collector
+distributions. These are OTel Collector distributions that are curated to
+Collector components that are specific to that vendor. They may be a combination
+of custom, vendor-developed components, and curated Collector Contrib
+components. Using vendor-specific distributions ensures that you are using just
+the Collector components that you need, again reducing overall bloat.
```

Comment:
You'd add a link to https://opentelemetry.io/ecosystem/distributions/

Reply:
Added!